triage/models/estimators.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.linear_model import LogisticRegression
+
+from triage.models.transformers import DsappCutOff
```

Comment:
Fixed in 72fee3a822cb3f4f646e7b358d2bc6f25d6af462 and 588d4b65a7efec49384214df42bbf0fefe7530be